examples/gallery/grid/grdview_surface.py

```diff
@@ -45,6 +46,7 @@ def ackley(x, y):
     surftype="s",
     cmap="roma",
     perspective="135/30",
```

Comment:
```suggestion
    perspective=[135, 30],  # Azimuth SouthEast (135°), at Elevation 30°
```
Looks awesome! Just one minor suggestion here to use a list when setting the perspective, and an inline comment.